### DISCUSSION TITLE
If you're going to mention scratchaddons.com, it should actually be up!

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [ScratchAddons/ScratchAddons#293] [Discussion]
If you're going to mention scratchaddons.com, it should actually be up! Please put it up!

**Comment by retronbv** [comment]
Yes plz

**Comment by jeffalo** [comment]
i'm no engineer, but doesn't it take effort to make a website and isn't w_l already working on getting hans' site ready? seems like a useless issue to be honest

**Comment by GrahamSH-LLK** [comment]
Ok, then.

**Comment by Hans5958** [comment]
> 
> 
> i'm no engineer, but doesn't it take effort to make a website and isn't w_l already working on getting hans' site ready? seems like a useless issue to be honest

It's not useless when the site is not even up yet but it is already linked. I think it is better to avoid putting it first to avoid confusion.

In the meantime, I have removed the link. Somebody may put it again when it is up.

also, why no .org @ WorldLanguages

**Comment by WorldLanguages** [comment]
It's now up :)
Feel free to register .org and redirect to .com if you wish :P
